code-challenges.test.js

```diff
@@ -1,49 +1,172 @@
+// // Output : ["yellow", "blue", "pink", "green"] 
+// ///Output : ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
 
+const colors1 = ["purple", "blue", "green", "yellow", "pink"]
+const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
 
-// b) Create the function that makes the test pass.
-
+ const shuffleColors = (array) => {
+   array.shift () 
+   for (let i = 0; i < array.length; i++) {
+    let shuffled = Math.floor(Math.random() * array.length)
+    let newArray = array[i] 
+    array[i]= array[shuffled]
+    array[shuffled] =newArray
+ }
+    return array 
+ }
+
+// PASS  ./code-challenges.test.js
+// shuffleColors
+//   ✓ takes in an array of colors, removes first item from array and shuffles the array of colors
+
+// Test Suites: 1 passed, 1 total
+// Tests:       1 passed, 1 total
+// Snapshots:   0 total
+// Time:        0.169 s, estimated 1 s
+// Ran all test suites.
 
 // --------------------2) Create a function that takes an array of numbers and returns an array of the minimum and maximum numbers in that order.
 
 // a) Create a test with expect statements for each of the variables provided.
 
 const nums1 = [3, 56, 90, -8, 0, 23, 6]
-// Expected output: [-8, 90]
+// // Expected output: [-8, 90]
 const nums2 = [109, 5, 9, 67, 8, 24]
 // Expected output: [5, 109]
 
+describe("minmumMaxium", () => {
+  it ("takes in an array of numbers and returns an array of the minimum and maxium numbers in that order", () => {
+       const nums1 = [3, 56, 90, -8, 0, 23, 6]
+       const nums2 = [109, 5, 9, 67, 8, 24]  
+       expect(minmunMaxium(nums1)).toEqual(expect.arrayMinmax([-8, 90]))
+       expect(minmunMaxium(nums2)).toEqual(expect.arrayMinmax([5, 109]))
+  })  
+ })
+//  FAIL  ./code-challenges.test.js
+//   minmumMaxium
+//     ✕ takes in an array of numbers and returns an array of the minimum and maxium numbers in that order
 
-// b) Create the function that makes the test pass.
+//   ● minmumMaxium › takes in an array of numbers and returns an array of the minimum and maxium numbers in that order
 
+//     ReferenceError: minmunMaxium is not defined
+
+// b) Create the function that makes the test pass.
+//Pseudo Code
+//Create a function 
+//The function will be called "minimumMaxium"
+//the Array will returned the smallest & biggest number order.  
+//Iterate the values in the array 
+//use the spread operator 
+//create variables for Math.min and Math.max
+//output: [-8,90]
+//output: [5, 109]
+// minimumMaximum= (array) 
+// var new_array = []
+// for (var i = 0; i < array.length; i++){
+// if (array[i] === Math.max.apply(Math, array)){
+//  new_array.unshift(array[i])
+//  else if (array[i] === Math.min.apply(Math, array)) 
+
+  const minimumMaximum =(arrary) => {
+    let minimum = Math.min(...arraryarray) 
+    let maximum = Math.max(...array)
+    let result = ([minimum, maximum])
+    return result
+  }
```

Comment:
Logic looks good here, however, there are a lot of small typos in this function and your test that will make this fail.  Make sure you are keeping the names of functions and parameters consistent.